modules/era/lua/dynamis/globals/era_dynamis.lua

```diff
@@ -0,0 +1,1185 @@
+--------------------------------------------
+--         Dynamis 75 Era Module          --
+--------------------------------------------
+--------------------------------------------
+--       Module Required Scripts          --
+--------------------------------------------
+require("scripts/mixins/job_special")
+require("scripts/globals/battlefield")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/npc_util")
+require("scripts/globals/status")
+require("scripts/globals/titles")
+require("scripts/globals/utils")
+require("scripts/globals/zone")
+require("scripts/globals/msg")
+require("scripts/globals/pathfind")
+require("modules/module_utils")
+--------------------------------------------
+--       Module Extended Scripts          --
+--------------------------------------------
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_bastok_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_beaucedine_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_buburimu_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_jeuno_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_qufim_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_san_d_oria_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_valkurm_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_windurst_mobs")
+require("modules/era/lua/dynamis/mob_spawning_files/dynamis_xarcabard_mobs")
+--------------------------------------------
+--       Module Affected Scripts          --
+--------------------------------------------
+require("scripts/globals/dynamis")
+--------------------------------------------
+
+local m = Module:new("era_dynamis")
+
+xi = xi or {}
+xi.dynamis = xi.dynamis or {}
+
+--------------------------------------------
+--        Global Dynamis Variables        --
+--------------------------------------------
+local dynamis_timeless = 4236
+local dynamis_perpetual = 4237
+local dynamis_min_lvl = 65
+local dynamis_reservation_cancel = 180
+local dynamis_reentry_days = 3
+local dynamis_rentry_hours = 71
```

Comment:
Yes these do, it is meant to allow for easy customization through the module rather than having settings spread between different luas. I made this entire lua work off of only settings available in the module luas with the exception of timeless_hourglass_price.